aziotctl/aziotctl-common/src/set_log_level.rs

```diff
@@ -0,0 +1,68 @@
+use std::fmt;
+use std::fs::File;
+use std::io;
+use std::io::prelude::*;
+use std::process::Command;
+use std::str::FromStr;
+
+use crate::ServiceDefinition;
+
+#[allow(clippy::module_name_repetitions)]
+/// # Errors
+///
+/// Will return `Err` if the systemd folder for the service does not exist or the user does not have
+/// permission to read it.
+pub fn set_log_level(services: &[&ServiceDefinition], level: LogLevel) -> Result<(), io::Error> {
+    for ServiceDefinition {
+        service,
+        sockets: _,
+    } in services
+    {
+        write_log(service, level)?;
+    }
+    Command::new("systemctl").arg("daemon-reload").output()?;
+
+    println!("Set log level to {} for all services. Run the `system restart` command for the changes to take effect.");
```

Comment:
"Run the `system restart` command" will make the user think they need to run `system restart` rather than `aziotctl system restart` I assume you made this generic because it'll have to work for `iotedge` too? Jet get `argv[0]` for that.

```rust
let program_name =
    std::env::args_os().next()
    .and_then(|arg| arg.into_string().ok())
    .unwrap_or_else(|| "<unknown>".to_owned());
```

Same for the others like the mention of "`system logs`" in the `system status` code below.